cli/engine/providers/aws/InfrastructureBuilder.py

```diff
@@ -220,26 +212,25 @@ def get_public_key(self):
         return public_key_config
 
     def add_security_rules_inbound_efs(self, infrastructure, security_group):
-        ags_allowed_to_efs = select_all(infrastructure, lambda item: item.kind == 'infrastructure/virtual-machine' and
+        vm_allowed_to_efs = select_all(infrastructure, lambda item: item.kind == 'infrastructure/virtual-machine' and
```

Comment:
You are using select_all here so:
```suggestion
        vms_allowed_to_efs = select_all(infrastructure, lambda item: item.kind == 'infrastructure/virtual-machine' and
```

Reply:
Fixed